.github/actions/test-server/action.yaml

```diff
@@ -37,6 +37,11 @@ outputs:
 runs:
   using: "composite"
   steps:
+    - name: Set container name
+      id: set-container-name
+      run: echo "container_name=$([[ '${{ inputs.jimm-version }}' == 'dev' ]] && echo 'jimm' || echo 'jimm-test')" >> $GITHUB_ENV
```

Comment:
Maybe we should update the docker compose so that both both profiles start jimm with the same name? Just `jimm`.